viewer/src/public/migration/Cognite3DModel.ts

```diff
@@ -487,8 +487,8 @@ export class Cognite3DModel extends THREE.Object3D implements CogniteModelBase {
    */
   async showNodeByTreeIndex(treeIndex: number, applyToChildren = false): Promise<number> {
     const treeIndices = await this.determineTreeIndices(treeIndex, applyToChildren);
-    this.hiddenNodes.delete(treeIndex);
-    this.cadNode.requestNodeUpdate([treeIndex]);
+    treeIndices.forEach(idx => this.hiddenNodes.delete(idx));
+    this.cadNode.requestNodeUpdate(treeIndices.asArray());
```

Comment:
That's not related to that PR at all, but I think that conversion of treeIndices to an array should be called `toArray` I guess that name is more common to what we have in js with its toString toNumber toLowercase and other similarly named functions.

Reply:
Agreed, fixed.